include/deal.II/integrators/advection.h

```diff
@@ -196,6 +202,84 @@ namespace LocalIntegrators
     }
 
     /**
+     * Advection residual operator in weak form
```

Comment:
Please describe this as 'scalar advection residual operator in weak form' to more clearly contrast with the vector-valued function.